stable/redis-ha/values.yaml

```diff
@@ -259,6 +259,11 @@ exporter:
   # Additional args for redis exporter
   extraArgs: {}
 
+  # Used to mount a LUA-Script via config map and use it for metrics-collection
+  # script:
+  #   configmap: prometheus-redis-exporter-script
```

Comment:
I think if we're going to do this, we need to have the chart create the ConfigMap and write the script as the value to the configmap so things can be set more statically. This make things easy for people to make mistakes. 

People that are uninformed may not understand they have to define their own ConfigMap template. 

I'm open to having my mind changed here.

Reply:
I like this idea, I had just copied over exactly what was being done in the prometheus-redis-exporter chart. I'll push a change with your suggestion, seems easier to use.

Reply:
@DandyDeveloper I have implemented your suggestion, let me know what you think.